src/sage/groups/abelian_gps/all.py

```diff
@@ -20,10 +20,11 @@
 #                  http://www.gnu.org/licenses/
 #*****************************************************************************
 
-#from dual_abelian_group import DualAbelianGroup
+# from dual_abelian_group import DualAbelianGroup
 from .abelian_group import AbelianGroup, word_problem
 from .values import AbelianGroupWithValues
 
-# TODO:
-# Implement group homset, conversion of generator images to morphism
+# .. TODO::
+
+#     Implement group homset, conversion of generator images to morphism
 from .abelian_group_morphism import AbelianGroupMorphism
```

Comment:
best not to touch the .all files -- this will conflict with the big refactoring PR #36676.

Also we don't really use RST directives in comments.